README.md

```diff
@@ -73,13 +106,23 @@ By default, HTTP status codes between 400 - 499 are logged at ERROR level.  You
 If you set `REQUEST_LOGGING_HTTP_4XX_LOG_LEVEL=logging.INFO` they will be logged the same as normal requests.
 ### REQUEST_LOGGING_SENSITIVE_HEADERS
 The value of the headers defined in this settings will be replaced with `'*****'` to hide the sensitive information while logging. By default it is set as `REQUEST_LOGGING_SENSITIVE_HEADERS = ["HTTP_AUTHORIZATION", "HTTP_PROXY_AUTHORIZATION"]`
+### DJANGO_REQUEST_LOGGING_LOGGER_NAME
+Name of the logger that is used to log django.request occurrances with the new LoggingMiddleware. Defaults to "django.request".
+### DJANGO_REQUEST_LOGGING_NO_LOGGING_DEFAULT_VALUE
+Global default to activate/deactivate logging of all views. Can be overruled for each individual view by using the @no_logging decator's "value" parameter.
+### DJANGO_REQUEST_LOGGING_LOG_HEADERS_DEFAULT_VALUE = False
```

Comment:
Please adapt the default value to `True`. Also for `DJANGO_REQUEST_LOGGING_LOG_BODY_DEFAULT_VALUE` and `DJANGO_REQUEST_LOGGING_LOG_RESPONSE_DEFAULT_VALUE`